src/test/ExternalExecutor.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Later on, these contracts used for testing should be placed in `./src/test/contracts`

Reply:
It is not trivial for our waffle configuration, we would need a separate one for test contacts. And for each previous EVM version we are testing agains as well